cdhweb/pages/models.py

```diff
@@ -223,9 +223,8 @@ def get_context(self, request):
             updates = BlogPost.objects.published().recent()[:3]
         context['updates'] = updates
 
-        # add up to 4 highlighted, published projects
-        projects = list(Project.objects.live().highlighted())
-        shuffle(projects)
+        # add up to 4 randomly selected highlighted, published projects
+        projects = list(Project.objects.live().highlighted().order_by("?"))
```

Comment:
fwiw, could be a one-liner now; you don't need the `list` anymore (it was needed for shuffling), and the `[:4]` can be applied directly to the queryset